engine/schema/src/main/resources/META-INF/db/schema-41810to41900.sql

```diff
@@ -605,3 +605,22 @@ CREATE TABLE `cloud`.`oauth_provider` (
   `removed` datetime COMMENT 'date removed if not null',
   PRIMARY KEY (`id`)
   ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- Via Java the configuration is inserted after the upgrade. Therefore, in order to derive the 'usage.timezone' value from `usage.aggregation.timezone' and
+-- `usage.execution.timezone', we need to insert the configuration manually.
+
+INSERT INTO `cloud`.`configuration` (category, `instance`, component, name, value, description, default_value, `scope`, is_dynamic)
```

Comment:
```suggestion
```
`ConfigKey<>` are automatically added to the DB.

Reply:
The upgrade script is executed before the management server adds configurations to the database. Here, I am manually adding `usage.timezone` to the database so I can derive its value from `usage.execution.timezone` or `usage.aggregation.timezone`.